index.html

```diff
@@ -3094,18 +3134,18 @@ <h3 id="applying">
           </li>
         </ol>
       </section>
-      <section>
+      <section data-dfn-for="imageresource">
```

Comment:
You shouldn't need:
```
data-dfn-for="imageresource"
```

ReSpec will find it from the parent section.

Reply:
Maybe it should, but it doesn't ... removing this and I get one more error